vignettes/articles/identifying-false-positives.Rmd

```diff
@@ -0,0 +1,409 @@
+
+The second common feature of false detections is that within a run a lot of bursts are missed. During the process of automated tag filtering up to 60 bursts can be missed without the run being terminated for the identified tag. This makes biological sense as the detection probability of a tag deployed on a moving animal also depends on the structure and type of the landscape, the orientation of the antenna of the tag as well as on the distance and height the animal is moving relative to the recording antenna. If the animal drops in height, vanishes behind a structure or if it quickly turns its body axis, recordings of the tag can be interrupted, i.e. bursts can be missed. Recordings with a very high amount of missed bursts, however, might be less trustworthy.
+
+The third feature is that the time intervals of the detected bursts might not properly match the time intervals  recorded during tag registration. If you calculate ‘measured’ burst intervals between two subsequently recorded bursts of the same tag, i.e. ignoring missed bursts, they sometimes deviate widely from the burst interval as specified by the manufacturer and tag registration. Many missed bursts will result in an average ‘measured’ burst interval substantially larger than the expected burst interval of the tag and may be indicative of a false positive.
+
+As pointed out, none of the three common features of false positives described above can alone describe the quality of a detection. We thus want to use all three of them to classify our detection dataset *a priori* into ‘good’ and ‘bad’ detections. 
+
+```{r, warning = FALSE, message = FALSE}
+
+# set system environment time zone to GMT
+Sys.setenv(tz="GMT")
+
+# load required packages
+library(motus)
+library(rms)
+library(ggplot2)
+library(dplyr)
+library(lubridate)
+
+# set base theme for plots
+theme_set(theme_bw(base_size = 18))
+
+### download sample data from James Bay Shorebird Project #176
+#   note: if you haven't downloaded sample dataset before, use 'new = T'
+#   login name and password are both 'motus.sample'
+#   specify 'dir =' to save elsewhere than working directory
+db <- tagme(176, new = F, update = F, forceMeta = F)
+
+t <- tbl(db, "alltags")
+
+df <- t %>% 
+  filter(!is.na(tagDeployID)) %>%                                              # remove tags with no deployment or burst interval
+  select(hitID, runID, ts, sig, sigsd, noise, freq, freqsd, slop, burstSlop,   # select variables
+         motusTagID, port, runLen, tagModel, tagBI, tagDeployID,
+         deviceID, recvDeployID, recvDeployLat, recvDeployLon, recv, recvSiteName) %>%
+  arrange(motusTagID, ts) %>%                                                  # order data frame
+  collect %>% as.data.frame %>%                                                # convert into flat data frame
```

Comment:
Best to use `()` when calling a function. And is there a specific reason for using as.data.frame() after? I don't think it's necessary unless you really want to avoid tibbles? I've been thinking about removing it from all the vignettes... what do you think?
```suggestion
  collect()                                                # convert into flat data frame
```